src/codegen/templates/_grpc_interpreter.py.mako

```diff
@@ -56,11 +57,16 @@ class GrpcStubInterpreter(BaseInterpreter):
     %endif
         response = self._invoke(
             self._client.${snake_to_pascal(func.function_name)},
-        %if (len(func.function_name) + len(get_grpc_interpreter_call_params(func, sorted_params))) > 68:
+        %if (len(func.function_name) + len(grpc_interpreter_params)) > 68:
             grpc_types.${snake_to_pascal(func.function_name)}Request(
-                ${get_grpc_interpreter_call_params(func, sorted_params) + ')' | wrap(16, 16)})
+                ${grpc_interpreter_params + ')' | wrap(16, 16)})
         %else:
-            grpc_types.${snake_to_pascal(func.function_name)}Request(${get_grpc_interpreter_call_params(func, sorted_params)+ ')'})
+            grpc_types.${snake_to_pascal(func.function_name)}Request(${grpc_interpreter_params + ')'})
+        %endif
+        %if (check_if_parameters_contain_read_array(sorted_params)):
+        % for param in get_read_array_parameters(func):
+        ${param}[:]=response.${param}
```

Comment:
Style: put spaces around `=` in assignments, omit spaces around `=` when declaring optional parameters.

(This seems to match Black's formatting.)